main.go

```diff
@@ -53,11 +53,11 @@ func main() {
 				},
 				cli.StringFlag{
 					Name:  "volume, v",
-					Usage: "Mount a volume into the cluster node (Docker notation: `source:destination`)",
+					Usage: "Mount one or more volumes into the cluster node (Docker notation: `source:destination[,source:destination]`)",
 				},
 				cli.StringFlag{
 					Name:  "version",
-					Value: "v0.3.0",
+					Value: "v0.4.0",
```

Comment:
Can get the latest version number dynamically from github instead of fixing it in the code.

Reply:
A fast shell/curl solution:
```
$ curl -s https://api.github.com/repos/rancher/k3s/releases/latest \
> | grep "tag_name" \
> | cut -d : -f 2
 "v0.4.0",
$
```

But it's better to do this by golang.

Reply:
Definitely way better than hardcoding it. I'll have a look on how to do it in golang or we just embed it in the build process.
Thanks for your feedback :)

Reply:
Take a look at this https://github.com/rancher/k3d/blob/6c072fb6d93d60e114fbc7a4ff68968af9768ddf/main.go#L19

Reply:
K3s version is now being grabbed from GitHub at build time.
I will add a `--latest` flag soon that will use a Go function to get the latest version at runtime.
This cannot be the default though since we should be able to run `k3d` without internet connection.